doc/cephfs/administration.rst

```diff
@@ -234,6 +234,45 @@ For example, to only allow Nautilus clients, use:
 
 Clients running an older version will be automatically evicted.
 
+Enforcing minimum version of CephFS client is achieved by setting required
+client features. Commands to manipulate required client features of a file
+system:
+
+::
+
+    fs required_client_features <fs name> add luminous
+    fs required_client_features <fs name> rm luminous
+
+To list all CephFS features
+
+::
+
+    fs feature ls
+
+
+Cephfs features and first release they came out.
+
++------------------+--------------+-----------------+
+| Feature          | Ceph release | Upstream Kernel |
++==================+==============+=================+
+| jewel            | jewel        | 4.5             |
++------------------+--------------+-----------------+
+| kraken           | kraken       | 4.13            |
++------------------+--------------+-----------------+
+| luminous         | luminous     | 4.13            |
++------------------+--------------+-----------------+
+| mimic            | mimic        | 4.19            |
++------------------+--------------+-----------------+
+| reply_encoding   | nautilus     | 5.1             |
++------------------+--------------+-----------------+
+| reclaim_client   | nautilus     | N/A             |
++------------------+--------------+-----------------+
+| lazy_caps_wanted | nautilus     | 5.1             |
++------------------+--------------+-----------------+
+| multi_reconnect  | nautilus     | 5.1             |
++------------------+--------------+-----------------+
+| deleg_ino        | octopus      | 5.6             |
```

Comment:
We need to make sure these are all described somewhere that makes sense to users/admins — I have no idea what the multi_reconnect feature is?

Reply:
multi_reconnect is:

If inodes in client cache can't be encoded in single reconnect messsage, client can split reconnect messages into multiple ones.

Reply:
I don't think that's explained to users anywhere, and anything showing up in this text should be.

Reply:
The problem is: these features (except reclaim_client) are more about mds->client protocol, not what user/admin cares about. It requires some knowledge of mds internal to understand what each feature is.